decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -49,6 +51,18 @@ def self.log_presenter_class_for(_log)
         Decidim::Elections::AdminLog::ElectionPresenter
       end
 
+      def real_time?
+        results_availability == "real_time"
+      end
+
+      def after_end?
+        results_availability == "after_end"
+      end
+
+      def per_question?
+        results_availability == "per_question"
+      end
+
```

Comment:
I see that you have an enum field above. You actually do not need these methods. 
You can use where you need: `results_real_time?`, `results_after_end?` etc ... 
If you want to use the short version, you could remove the prefix from enum. 
```suggestion
```

Reply:
good point